host/src/PrelaunchApp/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace App
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine($".NET version:{RuntimeInformation.FrameworkDescription}");
```

Comment:
could we do a bit more here. like process a json file or something? the more we do the more likelihood assemblies will be read from disk.

Reply:
Updated in the new iteration use some APIs/Types from framework assemblies.